project-extender/src/providers/jira-server-provider/JiraServerProvider.ts

```diff
@@ -1,11 +1,27 @@
 /* eslint-disable max-classes-per-file */
 /* eslint-disable class-methods-use-this */
 import JiraApi from "jira-client"
+import { fetch } from "cross-fetch"
 import { ProviderApi, ProviderCreator } from "../base-provider"
+import { Project } from "../base-provider/schema"
 
 class JiraServerProvider implements ProviderApi {
   provider: JiraApi | undefined = undefined
 
+  private requestBody: {
+    url: string
+    username: string
+    password: string
+  }
+
+  constructor(requestBody: {
+    url: string
+    username: string
+    password: string
+  }) {
+    this.requestBody = requestBody
```

Comment:
Like the comment below. I don't get why do we need to call the jira api directly when we have the library. Let's stick to one approach.